app/controllers/all/opening/projects_controller.rb

```diff
@@ -15,7 +15,7 @@ def confirmed_next_month
   def revised
     authorize Project, :index?
 
-    @projects = ByMonthProjectFetcherService.new.revised(month, year)
+    @projects = Conversion::Project.significant_date_revised_from(month, year)
```

Comment:
Why change this line, the action of `ByMonthProjectFetcherService.new.revised` has changed so it's not necessary